.appveyor.yml

```diff
@@ -0,0 +1,86 @@
+skip_branch_with_pr: true
+version: '{build}'
+image: Visual Studio 2017
+
+platform:
+  - x64
+
+# cache:
```

Comment:
I am not good with appveyor caching so not caching for now. I am not sure how to not install rtools and R when cached versions are found. But I guess its not that bad, the bottleneck is compiling cmdstan anyways.

Reply:
That's fine. We can maybe add caching later but not a big deal either way